src/Samples/Common/Views/ControlSamples/HierarchyRepeater/Basic.dothtml

```diff
@@ -0,0 +1,58 @@
+﻿@viewModel DotVVM.Samples.Common.ViewModels.ControlSamples.HierarchyRepeater.BasicViewModel, DotVVM.Samples.Common
+@masterPage Views/Samples.dotmaster
+
+<dot:Content ContentPlaceHolderID="Main">
+    <h1>Basic usage of <code>HierarchyRepeater</code></h1>
+
+    <h2>Server Rendering</h2>
+    <dot:HierarchyRepeater DataSource="{value: Roots}"
+                           ItemChildrenBinding="{value: Children}"
+                           LevelTagName="ul"
+                           Level:class="test"
+                           ItemTagName="li"
+                           RenderSettings.Mode="Server"
+                           PostBack.Update="true">
+        <span RenderSettings.Mode="Client" title={value: _root.GlobalLabel}>{{value: Name}}</span>
+        <dot:Button Click="{command: _root.ClickNode(_this)}" Text="{value: ClickCount}" title={value: $"{_root.GlobalLabel}: {Name}"} />
+    </dot:HierarchyRepeater>
+
+    <h2>Client Rendering</h2>
+    <dot:HierarchyRepeater DataSource="{value: Roots}"
+                           ItemChildrenBinding="{value: Children}"
+                           LevelTagName="ul"
+                           ItemTagName="li"
+                           RenderSettings.Mode="Client">
+        <span title={value: _root.GlobalLabel}>{{value: Name}}</span>
+        <dot:Button Click="{command: _root.ClickNode(_this)}" Text="{value: ClickCount}" />
+    </dot:HierarchyRepeater>
+
+    <h2>No Item and Level Tags</h2>
+    <dot:HierarchyRepeater DataSource="{value: Roots}"
+                           ItemChildrenBinding="{value: Children}"
+                           RenderSettings.Mode="Client">
+        <p title={value: _root.GlobalLabel}>{{value: Name}}</p>
+    </dot:HierarchyRepeater>
+
+    <h2>Empty</h2>
+    <dot:HierarchyRepeater DataSource="{value: Empty}"
```

Comment:
It would be nice to have a test without ItemTagName and LevelTagName to test that it also works with the placeholder

Reply:
Oh, yeah, sorry, my bad.